model/metabolites.tsv

```diff
@@ -2674,10 +2674,10 @@ mets	metsNoComp	metBiGGID	metKEGGID	metHMDBID	metChEBIID	metPubChemID	metLipidMa
 "MAM01613g"	"MAM01613"	"core8"	""	""	""	""	""	""	""	"core8"	"MNXM16842"	"m01613g"	"m01613g"
 "MAM01613c"	"MAM01613"	"core8"	""	""	""	""	""	""	""	"core8"	"MNXM16842"	"m01613c"	"m01613c"
 "MAM01613s"	"MAM01613"	"core8"	""	""	""	""	""	""	""	"core8"	"MNXM16842"	"m01613s"	"m01613s"
-"MAM01614c"	"MAM01614"	"crtstrn"	""	""	"CHEBI:101161"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614c"	"m01614c"
-"MAM01614m"	"MAM01614"	"crtstrn"	""	""	"CHEBI:101161"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614m"	"m01614m"
-"MAM01614r"	"MAM01614"	"crtstrn"	""	""	"CHEBI:101161"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614r"	"m01614r"
-"MAM01614s"	"MAM01614"	"crtstrn"	""	""	"CHEBI:101161"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614s"	"m01614s"
+"MAM01614c"	"MAM01614"	"crtstrn"	""	""	"CHEBI:16827"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614c"	"m01614c"
+"MAM01614m"	"MAM01614"	"crtstrn"	""	""	"CHEBI:16827"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614m"	"m01614m"
+"MAM01614r"	"MAM01614"	"crtstrn"	""	""	"CHEBI:16827"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614r"	"m01614r"
+"MAM01614s"	"MAM01614"	"crtstrn"	""	""	"CHEBI:16827"	""	"LMST02030092"	""	""	"crtstrn"	"MNXM162953;MNXM859"	"m01614s"	"m01614s"
```

Comment:
To address the current conflict with `develop`, could you please change `MAM01614s` (in the first column) to `MAM01614e`

Reply:
done, please see https://github.com/SysBioChalmers/Human-GEM/pull/358/commits/041ad028d203e6015589d8e901f2bf0ea07bb0be